mypyc/ir/ops.py

```diff
@@ -1412,6 +1412,55 @@ def accept(self, visitor: 'OpVisitor[T]') -> T:
         return visitor.visit_load_mem(self)
 
 
+class SetMem(Op):
+    """Write a memory location.
+
+    *(type *)src = val
```

Comment:
This is a bit confusing, since `src` means a different thing compared tot he comment below. I'd suggest writing it like this instead:
```
*(type *)dest = src
```